FueledUtils/LoadingState.swift

```diff
@@ -38,11 +68,27 @@ public enum LoadingState<Error: Swift.Error> {
 }
 
 public extension Action {
-	public func loadingState() -> SignalProducer<LoadingState<Error>, NoError> {
-		let loading = self.isExecuting.producer
+	///
+	/// **Deprecated**: Please use `getSafely(at:)` instead.
+	///
+	/// Refer to the documentation for `getSafely(at:)` for more info.
+	///
+	@available(*, deprecated, renamed: "loadingState")
+	// The unused parameter allows to bypass the compiler error "Invalid redeclaration of 'loadingState'",
+	// while retaining backward compatibility
+	public func loadingState(_ unused: Void = ()) -> SignalProducer<LoadingState<Error>, NoError> {
+		return self.loadingState.producer
```

Comment:
This is not the "same" producer as the deprecated one. If a binding occurs in the moment of action's executing the `loading` state will not be sent because this producer will emit the same events as the original signal. (see my comment for the new `var loadingState: Signal<LoadingState<Error>, NoError>` implementation)